tests/test_config_flow.py

```
@@ -0,0 +1,1334 @@
+                ):
+                    """Test async_step_verify_mfa shows form if no user_input is provided."""
+                    flow._client = MagicMock()
+                    flow._data = {CONF_USER_NAME: "user"}
+                    result = await flow.async_step_verify_mfa()
+                    assert result["type"] == "form"
+                    assert result["step_id"] == "verify_mfa"
+                    assert "errors" in result
+
+
+@pytest.mark.asyncio
+async def test_get_device_success(hass):
```

Ruff (0.12.2)

Redefinition of unused `test_get_device_success` from line 589

(F811)